### ISSUE TITLE
[BUG][JavaSpring] description value for @Operation annotation are not generated from OpenAPI definition

### ISSUE THREAD

**Issue by pichlau** [OpenAPITools/openapi-generator#11779]
#### Bug Report Checklist

- [x] Have you provided a full/minimal spec to reproduce the issue?
- [ ] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [x] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [x] Have you searched for related issues/PRs?
- [x] What's the actual output vs expected output?
- [ ] [Optional] Sponsorship to speed up the bug fix or feature request ([example](https://github.com/OpenAPITools/openapi-generator/issues/6178))

##### Description

When I run the JavaSpring generator in my maven build execution from my OpenAPI definition yaml file, in the java code generated the #description value in the @Operation annotation is absent.

    @Operation(
        operationId = "xxxxxx",
        summary = "xxxxxxx",
        tags = { "xxxxx" },
    )

it should be:

    @Operation(
        operationId = "xxxxxx",
        summary = "xxxxxxx",
        description = "xxxxxxxx",
        tags = { "xxxxx" },
    )

##### openapi-generator version
5.4.0

##### OpenAPI declaration file content or url

Any endpoint definition with a description will reveal the problem.

##### Generation Details

Generation running with maven plugin.

##### Steps to reproduce

Clear

##### Related issues/PRs

Clear

##### Suggest a fix

To correct the problem, simply replace in "modules/openapi-generator/src/main/resources/JavaSpring/api.mustache"

    @Operation(
        operationId = "{{{operationId}}}",
        {{#summary}}
        summary = "{{{.}}}",
        {{/summary}}
        {{#vendorExtensions.x-tags.size}}

by

    @Operation(
        operationId = "{{{operationId}}}",
        {{#summary}}
        summary = "{{{.}}}",
        {{/summary}}
        {{#description}}
        description= "{{{.}}}",
        {{/description}}
        {{#vendorExtensions.x-tags.size}}

**Comment by abarstow** [comment]
yeah, I noticed this and my local hack for `modules/openapi-generator/src/main/resources/JavaSpring/api.mustache` is:
```
<         {{#description}}
<         description= "{{{.}}}",
<         {{/description}}
---
> 
>         description = "{{{notes}}}",
> 
```

**Comment by pichlau** [comment]
Can this fix be added in the #6.0.x and #master branches please.
